Sources/Models/BlogPosts/BlogPost0001_Announcement.swift

```diff
@@ -10,7 +10,7 @@ even deeper into functional programming, showing real world use cases and more!
   contentBlocks: [
     .init(
       content: """
-      It's hard to believe it's been nearly 3 months since we launched [Point-Free](\(url(to: .home))), but
+      It's hard to believe it's been nearly 3 months since we launched [Point-Free](/), but
```

Comment:
since the router no longer lives in the same module as the models we can't use it, so I've hard coded the url's.

I don't think this is a huge deal since 1) these url's shouldn't change, we should only add new url's, and 2) eventually we probably want to move to have transcripts as markdown that we parse rather than embedded in swift files, and in that case we wouldn't have access to the router anyway.